__tests__/db.test.js

```diff
@@ -114,3 +114,70 @@ describe("GET /api/articles:id", () => {
       })
   })
 })
+
+describe("/api/articles/:article_id/comments", () => {
+  test("200: Returns with an array of comments", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comments).toBeInstanceOf(Array)
+      })
+  })
+  test("200: Comments possess their required keys", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comments.length).toBeGreaterThan(0)
+        body.comments.forEach((comment) => {
+          expect(comment.comment_id).toEqual(expect.any(Number))
+          expect(comment.votes).toEqual(expect.any(Number))
+          expect(comment.created_at).toEqual(expect.any(String))
+          expect(comment.author).toEqual(expect.any(String))
+          expect(comment.body).toEqual(expect.any(String))
+          expect(comment.article_id).toEqual(expect.any(Number))
+        })
+      })
+  })
+  test("200: Returned comments are of the correct article", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comments.length).toBeGreaterThan(0)
+        body.comments.forEach((comment) => {
+          expect(comment.article_id).toBe(1)
```

Comment:
I see you have done this here, I would just incorporate this into your previous test